client/src/components/CommentApp/comments.js

```diff
@@ -14,6 +14,9 @@ function initComments() {
     window.commentApp.renderApp(
       commentsElement, commentsOutputElement, data.user, data.comments, new Map(Object.entries(data.authors)), STRINGS
     );
+
+    // eslint-disable-next-line @typescript-eslint/no-use-before-define
+    document.querySelectorAll('[data-comment-add]').forEach(initAddCommentButton);
```

Comment:
If we're using this attribute to initialise field level widgets, we should remove it from the draftail control and use a different data-attribute to ensure clicking it doesn't remove focus.

Reply:
Good spot! Changed it here